src/index.jsx

```diff
@@ -52,8 +52,8 @@ const injectBarInDOM = (data) => {
     if (visible) {
       barNode.dataset.drawerVisible = true
     } else {
-      // total drawer closing transition duration = .5s
-      setTimeout(() => { barNode.dataset.drawerVisible = false }, 500)
+      // total drawer closing transition duration = .3s
+      setTimeout(() => { barNode.dataset.drawerVisible = false }, 300)
```

Comment:
Not a big fan on relying on a timeout related to a CSS duration, both values may evolve differently. Instead I suggest to use a TransitionListener. But it will be for a next PR. :)